packages/gnome_settings_daemon.rb

```diff
@@ -1,6 +1,6 @@
-require 'package'
+require 'buildsystems/meson'
 
-class Gnome_settings_daemon < Package
+class Gnome_settings_daemon < Meson
   description 'GNOME Settings Daemon'
   homepage 'https://gitlab.gnome.org/GNOME/gnome-settings-daemon'
   version '43.0'
```

Comment:
Update to [45.1](https://gitlab.gnome.org/GNOME/gnome-settings-daemon/-/tags/45.1)?

Reply:
No longer builds w/o systemd.

Reply:
deprecating.